azure/scope/cluster.go

```diff
@@ -35,7 +35,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	infrav1 "sigs.k8s.io/cluster-api-provider-azure/api/v1alpha3"
-	azure "sigs.k8s.io/cluster-api-provider-azure/cloud"
+	azure "sigs.k8s.io/cluster-api-provider-azure/azure"
```

Comment:
```suggestion
	"sigs.k8s.io/cluster-api-provider-azure/azure"
```

Please do this for the rest of the imports.